app/services/user/bulk_destroy.rb

```diff
@@ -5,10 +5,13 @@ class User::BulkDestroy
 
   params do
     attribute :user_ids, :array
+    attribute :block_ip_and_email, :boolean
 
     validates :user_ids, length: { maximum: 100 }
   end
 
+  options { attribute :actor_ip }
```

Comment:
In the case we can’t drop this, can we move it to the top, please? As `options` isn’t really part of the execution flow (it’s not validating anything, it can’t fail, etc.), it’s better to have it as the first DSL keyword.